src/TBuffer.h

```diff
@@ -193,9 +193,13 @@ class TBuffer
 
 
     std::deque<TChar> bufferLine;
+    // stores the sequence of TChars that make up a line in a buffer
     std::deque<std::deque<TChar>> buffer;
-    QStringList timeBuffer;
+    // stores the actual content of lines
     QStringList lineBuffer;
+    // stores timestamps associated with lines
+    QStringList timeBuffer;
+    // stores a boolean whenever the line is a prompt one
```

Comment:
💡 In the future I may revise this to be a `QList<QFlags>` type member to hold per line information about that line, I am particularly thinking about flags (i.e. binary 2^n enums) for:
* whether this is a prompt line
* whether the line contains a match for the current search term - as part of improving the current irreversible highlighting of searches in the main console...
* whether the line contains any blinking text - so that it can be picked out to be redrawn when the next slow/fast blinking phase clock change occurs

Reply:
Please move the 💡 to an issue of its own so it doesn't drown in the comment section of a closed PR and can be pondered more in due time.

Reply:
Given the time currently available to me I don't feel that raising an issue for this is worthwhile at this point. I was just pointing out a potential internal feature/development for the future...